protocol/migrations/sqlite/1695331492_add_status_link_previews.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE user_messages ADD COLUMN unfurled_status_links BLOB;
```

Comment:
Just sharing, nothing to do now, but I still fear status links have far more reasons to change than the relatively stable API of normal unfurled links, which could mean painful migrations. But since the `user_messages` table should never have its data migrated (according to our past discussions) then things can get tricky in the future.

A part of me still thinks this more "fragile" blob should be captured in a separate table, with explicit columns, although the join could affect the performance (or maybe not).

Don't worry, I'm just rambling here.